config/cmake/scripts/HDF5config.cmake

```diff
@@ -24,9 +25,9 @@ cmake_minimum_required (VERSION 3.18)
 #            MinGW     * MinGW Makefiles
 #            Unix      * Unix Makefiles
 #            VS2022    * Visual Studio 17 2022
-#            VS202264  * Visual Studio 17 2022
+#            VS202264  * Visual Studio 17 2022 Win64
```

Comment:
No that Win64 is not correct for VS2022 and VS2019

Reply:
What do you mean? 
What's the difference between VS2022 and VS202264? 
Is VS202264 redundant?

Reply:
@byrnHDF, I don't understand what you're saying.
I'll close this PR.
You can delete comments and create another PR.